src/Stratis.Bitcoin/P2P/Peer/NetworkPeer.cs

```diff
@@ -457,8 +506,8 @@ public bool IsConnected
         /// <summary>Version message payload received from the peer.</summary>
         public VersionPayload PeerVersion { get; private set; }
 
-        /// <summary>Set to <c>1</c> if the peer disconnection has been initiated, <c>0</c> otherwise.</summary>
-        private int disconnecting;
+        /// <summary>Set to <c>1</c> if the peer disconnection has been initiated, <c>0</c> otherwise.</summary> 
```

Comment:
Consider changing the comment as well. If you changed 'disconnecting' to 'disconnected' then probably comment should be like: 

`Set to <c>1</c> if the peer is disconnected, <c>0</c> otherwise.`

Reply:
I actually think the previous name was a mistake. The value of 1 does only mean that somewhere in the past it has been initiated. It does not mean the action was still in progress. So the comment was right, just the name of the variable seemed wrong.